packages/story-editor/src/utils/inRange.ts

```diff
@@ -13,6 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const inRange = (value, { MIN, MAX }) => value >= MIN && value <= MAX;
+
+interface minMaxProps {
+  MIN: number;
+  MAX: number;
+}
+
+const inRange = (value: number, { MIN, MAX }: minMaxProps) =>
```

Comment:
While we're at it:

This function is only ever used in the `SlugPanel` component and it's basically a single line of code.

Let's just delete this file and define that function inline in that component's file.